.gitattributes

```diff
@@ -0,0 +1,2 @@
+# Add syntax highlighting to NGINX config files
+nginx.*.conf linguist-language=Nginx
```

Comment:
I'm not sure how many wildcards this will take, but making it `nginx*.conf*` would also catch [nginx.conf.jinja](https://github.com/python/psf-salt/blob/d25f5bf1e001fe2bc8ebe59ea0ee2d24fcf4846a/salt/buildbot/config/nginx.conf.jinja), for example.

Reply:
would that make us lose Jinja highlighting?

Reply:
I don't think so, because AFAICT we don't currently have *any* highlighting there :)

Reply:
Ah, I did find a counterexample in [haproxy/config/nginx-redirect.conf.jinja](https://github.com/python/psf-salt/blob/d25f5bf1e001fe2bc8ebe59ea0ee2d24fcf4846a/salt/haproxy/config/nginx-redirect.conf.jinja), so maybe my suggestion is not a good one (though it would not actually catch that particular one).